test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -122,7 +144,7 @@ TEST_F(XdsOverrideHostTest, NoConfigReportsError) {
       absl::InvalidArgumentError("Missing policy config"));
 }
 
-TEST_F(XdsOverrideHostTest, ConfigRequiresChildPolicy) {
+TEST_F(XdsOverrideHostTest, ValidateChildPolicyConfig) {
```

Comment:
Suggest moving the config tests to their own file, since they don't require any of the other infrastructure being set up in this test.  For an example of an LB policy config test, see https://github.com/grpc/grpc/blob/master/test/core/client_channel/outlier_detection_lb_config_parser_test.cc.